python/ray/dashboard/modules/metrics/dashboards/default_dashboard_panels.py

```diff
@@ -63,6 +63,22 @@ def max_plus_pending(max_resource, pending_resource):
             ),
         ],
     ),
+    Panel(
+        id=38,
+        title="Running Tasks by Name",
+        description="Current number of (running) tasks with a particular name. Task resubmissions due to failures or object reconstruction are shown with (retry) in the label.",
+        unit="tasks",
+        targets=[
+            Target(
+                expr='sum(ray_tasks{{IsRetry="0",State=~"RUNNING_*",{global_filters}}}) by (Name)',
+                legend="{{Name}}",
```

Comment:
will this regex capture the `RUNNING` state?

Reply:
good catch, I'll change to `RUNNING*`